### PR TITLE
Bug 1407380 - Add error message on python crash

### PR DESCRIPTION
**Pull request by Dyex719**
Add a helpful error message

### CODE REVIEW

**Comment by Dyex719** [comment]
I think I understand, thank you for the idea!

File: scriptworker/test/test_task.py [hunk 1]
```diff
@@ -150,6 +150,23 @@ def test_reclaim_task_non_409(context, successful_queue, event_loop):
 
 
 @pytest.mark.asyncio
+async def test_run_task_negative_11(context, mocker):
```
Comment (escapewindow) [inline scriptworker/test/test_task.py]: We lost the `@pytest.mark.asyncio` for the next test below with this change. I pushed a fix for this, just moved this up. Thank you!

**Comment by escapewindow** [comment]
Fixes #164.

**Comment by escapewindow** [comment]
Thank you!

File: scriptworker/test/test_task.py (hunk 1 above)
Comment (Dyex719) [reply scriptworker/test/test_task.py]: oops. Sorry for the mix up. You're welcome :)

**Pull request merged**